frontend/styles/var/breakpoints.scss

```diff
@@ -1,12 +1,5 @@
-$break-xxl: 1600px;
-$break-xl: 1500px;
-$break-lg: 1400px;
-$break-md: 990px;
-$break-sm: 880px;
-$break-xs: 768px;
-$break-mobile-lg: 650px;
-$break-mobile-md: 576px;
-$break-mobile-sm: 480px;
-$break-mobile-xs: 320px;
-$tooltip-break-md: 1000px; // Prevents horizontal scrolling off viewport
-$table-controls-break: 1150px;
+$break-xl: 1440px;
+$break-lg: 1280px;
+$break-md: 1024px;
+$break-sm: 768px;
+$break-xs: 480px;
```

Comment:
Strictly limiting breakpoints will help keep responsive complexity in check as we grow. 

The idea is to not think of responsive as a series of breakpoints, but to build layouts that naturally scale to fit their containers using percentages. For example, avoiding hard-coding widths and using `max-width` with `width: 100%`, instead. That way, the width doesn't have to be updated at each breakpoint, and the element will naturally grow to fit its container.

Reply:
like cutting down the breakpoints. these seems like a good set to me.